crates/oxc_transformer/src/helpers/ast_helper.rs

```diff
@@ -0,0 +1,43 @@
+#[macro_export]
+macro_rules! ast_helper {
+    ($allocator: expr, $source: ident) => {{
```

Comment:
does `$source` need to be an `ident`? couldn't it be a `'static str`?

Reply:
I may think it can't be for dynamic strings, e.g. for defining property.